packages/website/src/features/Packages/Function.tsx

```diff
@@ -132,10 +124,13 @@ export const Function: FC<{
         // console.error(e);
         // setError(e?.message || e?.error?.message || e?.error || e);
         try {
+          /*
```

Comment:
shoudl we just get rid of this code if we have established its not needed? if it was needed for osmething, should we get this filled in?